calculator.js

```diff
@@ -0,0 +1,11 @@
+const billTotal = document.getElementById("bill-total");
+const submitButtom = document.getElementById("submit");
+const serviceSlider = document.getElementById("service-quality");
+const tipTotal = document.getElementById("tip-total");
+
+submitButtom.addEventListener("click", () => {
+  tipTotal.innerText = `Your tip should be: $${(
```

Comment:
I think this is an awesome way to do the tips. I really like the slider that goes by emoji mood.